benchmark/kvbench/test/storage_backend.py

```
@@ -0,0 +1,609 @@
+                },
+            )
+
+            logger.debug(
+                "Prepared sharded storage: tp=%d, rank=%d, %d shards x %d bytes",
+                tp_idx, rank, len(shard_fds), shard_size,
+            )
+
+        self._handles[key] = handle
+        return handle
+
+    def _create_chunked_descs(self, fd, file_offset, total_size, buffer):
```

Ruff (0.15.13)

[warning] Missing return type annotation for private function `_create_chunked_descs`

(ANN202)